packages/scope-sdk/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Scope Oracle SDK",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "version": "1.0.101",
+  "version": "1.0.102",
```

Comment:
The versioning is done automatically by Lerna (monorepo packaging tool), you need to undo this change (it increases the version automatically when you release the packages).